core/editor/src/tools/crop.rs

```diff
@@ -0,0 +1,12 @@
+use crate::events::Event;
+use crate::tools::Tool;
+use document_core::Operation;
+
+#[derive(Default)]
+pub struct Crop;
+
+impl Tool for Crop {
+	fn handle_input(&mut self, event: Event) -> Option<Operation> {
```

Comment:
If I understand this correctly, tools won't respond with an Operation. They will end up needing direct access to make calls to the Dispatcher with zero, one, or many events (defined in the lambda function in each state of the FSM).

Reply:
What do you mean?

Reply:
We discussed this further in Discord and concluded that we should indeed stick with returning an Operation, although a Vec of Operations instead of just 1 or 0. See https://github.com/GraphiteEditor/Graphite/pull/58#discussion_r605509501